src/Commands/QuantDrushCommands.php

```diff
@@ -70,14 +70,29 @@ private function getDrushPath() {
    * @option threads
    *   Number of threads to use (default 5)
    * @usage quant:run-queue --threads=5
+   * @option bail
+   *  Bail out of a seed run if another run is in progress. (default is false)
+   * @usage quant:run-queue --bail
    */
-  public function message($options = ['threads' => 5]) {
+  public function message($options = ['threads' => 5, 'bail' => false]) {
```

Comment:
I recommend to have `'bail' => true` by default, otherwise we risk taking too much server CPU/RAM resources when command executed frequently, such as every 5 minutes via a cron job.